diff --git a/src/jsc/bindings/windows/rescle.cpp b/src/jsc/bindings/windows/rescle.cpp
index a3171b4..1eb7b31 100644
--- a/src/jsc/bindings/windows/rescle.cpp
+++ b/src/jsc/bindings/windows/rescle.cpp
@@ -112,7 +112,7 @@ inline T round(T value, int modula = 4)
 std::wstring ReadFileToString(const wchar_t* filename)
 {
     std::wifstream wif(filename);
-    wif.imbue(std::locale(std::locale::empty(), new std::codecvt_utf8<wchar_t>));
+    wif.imbue(std::locale(std::locale::classic(), new std::codecvt_utf8<wchar_t>));
     std::wstringstream wss;
     wss << wif.rdbuf();
     return wss.str();
